packages/esm-ohri-pmtct-app/src/views/child-health/tabs/infant-postnatal-care.component.tsx

```diff
@@ -33,10 +35,13 @@ const InfantPostnatalList: React.FC<InfantPostnatalListProps> = ({ patientUuid }
   }, []);
 
   async function fetchMotherName() {
+    let motherName = '--';
     const response = await fetchPatientRelationships(patientUuid);
     if (response.length) {
+      motherName = response[0].personA.display;
       setMotherName(response[0].personA.display);
```

Comment:
Hi @ODORA0 just for clean up purposes, we no longer need to the useState hook for setting the mother name.

Reply:
@kajambiya Done